test/shared/src/main/scala/zio/test/PrettyPrint.scala

```diff
@@ -1,41 +1,30 @@
 package zio.test
 
 import zio.internal.ansi.AnsiStringOps
-import zio.{Chunk, NonEmptyChunk}
 import zio.stacktracer.TracingImplicits.disableAutoTrace
+import zio.{Chunk, NonEmptyChunk}
 
 /**
  * PrettyPrint will attempt to render a Scala value as the syntax used to create
  * that value. This makes it easier to copy-paste from values printed to the
  * console during tests back into runnable code.
  */
 private[zio] object PrettyPrint extends PrettyPrintVersionSpecific {
-  private val maxListLength = 10
   def apply(any: Any): String =
     any match {
-      case nonEmptyChunk: NonEmptyChunk[_] =>
-        prettyPrintIterator(nonEmptyChunk.iterator, nonEmptyChunk.size, "NonEmptyChunk")
-      case chunk: Chunk[_] =>
-        prettyPrintIterator(chunk.iterator, chunk.size, "Chunk")
-      case array: Array[_] =>
-        prettyPrintIterator(array.iterator, array.length, "Array")
+      case null => "<null>"
```

Comment:
I re-organise these cases to try to put the most probable types first.

`NonEmptyChunk` is IMO not the most probable type, for example